src/scss/pages/blog.module.scss

```
@@ -6,33 +6,20 @@
   @extend .enumerate-contents-wrapper;
 }
 
-.blog {
-  width: 100%;
-}
-
-.blog > h1 {
-  text-align: center;
-}
-
 .cardText {
-  @extend .text;
-}
-
-.pageLink::before {
-  white-space: pre;
-  content: "\A";
+  @include details.flex-style(wrap, bottom);
```

```suggestion
  @include details.flex-style(wrap, flex-start);
  flex-direction: column;
  align-items: flex-start;
```
（もしかしたら ⬇ こう書くほうがいいかもしれないですけど...）
```css
  display: flex;
  flex-direction: column;
  align-items: flex-start;
```

---

cc11f67 で修正